aks/application/variables.tf

```diff
@@ -104,6 +104,20 @@ variable "probe_path" {
   description = "Path for the liveness and startup probe. The probe can be disabled by setting this to null."
 }
 
+variable "prometheus_monitoring" {
+  type = object({
+    port   = string,
+    path   = string,
+    scrape = string
+  })
+  default = {
+    scrape = "false"
+    path   = ""
+    port   = ""
+  }
+  description = "parameters for prometheus monitoring"
```

Comment:
can we add a bit more to the description? what this is for and when you would set to true, etc.

Reply:
done